cmd/minishift/cmd/root.go

```diff
@@ -123,6 +126,10 @@ var RootCmd = &cobra.Command{
 			}
 		}
 
+		if hasEnabledExperimental {
+			fmt.Println("!!! Experimental features are enabled !!!")
```

Comment:
I think if we were to do something like this, we would need to do this where we process the options and print the warning when an actual experimental option is used.